### ISSUE TITLE
Provide a csv of sketches that don't have `alt text` or `redbubble links`

### ISSUE THREAD

**Issue by jonohey** [jonohey/sketchplanations#266]
I'd like to add alt text to all decent sketches and also see which ones I could usefully add prints for.

It would be great to get a csv export of all the data about each of the sketches from the API. You did this once before with title, URL, copy etc

I don't know if alt text is more difficult to fetch or not. I was considering paying someone to add the missing ones for me.

**Comment by jonohey** [comment]
Provided script and sample csv. I amended to add 'Published at' date also. data.mjs script file and documentation to run it using csv-writer stored in Google Drive.
